app/controllers/sessions_controller.rb

```diff
@@ -3,14 +3,27 @@ def new
   end
 
   def create
-    user = User.find_by(email: params[:session][:email])
-    if user && user.authenticate(params[:session][:password])
-      session[:user_id] = user.id
-      flash[:success] = "Successfully logged in!"
-      redirect_to(:back)
+    if Admin.find_by(email: params[:session][:email])
```

Comment:
The methods, they have been broken out. SO BROKEN OUT.

![image](https://cloud.githubusercontent.com/assets/3011748/5889748/cc17b8e0-a3f3-11e4-97c9-c83de3e24eb9.png)